OasisDEX/oasis-borrow
# DPM positions on owner page

# [DPM positions on owner page](https://app.shortcut.com/oazo-apps/story/6939/fetch-data-for-aave-multiply-position-in-my-position-view)

## Not Included

- automation handling / displaying
- correct net value

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- created `aaveDpmPositions$` observable which returns information about dpm positions on aave
- read from position-created events
- moved few observables from `AaveContext` to `AppContext` as they were required there
  
## How to test 🧪
  <Please explain how to test your changes>

- create some DPM positions.  see them on the page listed as multiply and earn.
